lib/engine/game/g_system18/map_neus_customization.rb

```diff
@@ -137,16 +155,31 @@ def map_neus_game_trains(trains)
           find_train(trains, '3')[:num] = 3
           find_train(trains, '4')[:num] = 2
           find_train(trains, '5')[:num] = 2
+          find_train(trains, '5')[:events] = [{ 'type' => 'close_companies' }]
           find_train(trains, '6')[:num] = 1
           find_train(trains, 'D')[:num] = 10
           trains
         end
 
         def map_neus_game_phases
-          self.class::S18_FULLCAP_PHASES
+          phases = self.class::S18_FULLCAP_PHASES
+          phases[1][:status] = %w[can_buy_companies]
+          phases[2][:status] = %w[can_buy_companies]
```

Comment:
Won't this mutate `S18_FULLCAP_PHASES`, which affects multiple titles? That could cause weird things when running tests across different titles, IIRC that's been a problem before. Unfortunately `.freeze` only freezes the top-level hash

Reply:
Good point.

Would `phases = self.class::S18_FULLCAP_PHASES.dup` work?

Reply:
Nevermind. I have a better way to fix it.